.github/workflows/binaries.yml

```diff
@@ -0,0 +1,118 @@
+name: Binaries Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-binaries:
+    name: Build ${{ matrix.binary }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            binary: fioctl-linux-amd64
+          - os: ubuntu-latest
+            binary: fioctl-linux-arm64
+            qemu: true
+            qemu-platform: arm64
+          # Let's build on the ubuntu to be faster
+          - os: ubuntu-latest
+            binary: fioctl-windows-amd64.exe
+          - os: macos-latest
+            binary: fioctl-darwin-amd64
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        if: matrix.qemu
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: ${{ matrix.qemu-platform }}
+
+      - name: Build Binary
+        run: |
+          if [ "${{ matrix.binary }}" == "fioctl-windows-amd64.exe" ]; then
+            make fioctl-windows-amd64
+          else
+            make ${{ matrix.binary }}
+          fi
```

Comment:
If we run in `Bash`, can we do something like the following? :

```
binary="${{ matrix.binary }}"
make ${binary%.exe}
```

On the off chance we ever get other windows binaries or rename them. This just chops off the `.exe` part, but it is a `bashism`

Reply:
I am looking at : https://github.com/foundriesio/fioctl/blob/main/.github/workflows/release.yml#L25-L27
Are we using this one to do the releases right?

So, I think we should only call make and not build in the matrix. 
It should work since we are working with cross-compilation.

Reply:
I changed that